### DISCUSSION TITLE
Mali GPU ERROR

### DISCUSSION THREAD

**Discussion by MRGalaxy507** [skyline-emu/skyline#797] [Q&A] [answered]
I have a problem with skyline because I have a redmi note 9T with a processor dimensity 800u GPU MALI-G57 MC3 because before the sonic mania was running at 20 fps and with the new update I improve twice but the problem is that the game only closes currently it runs at 60 fps but it closes can you give me an explanation thanks keep it up you do an excellent job a hug from Latin America

**Comment by PixelyIon** [comment, marked as answer]
We know about this problem and have investigated it, a fix will come with the in-memory shader cache. There's an issue about this already that you can just track for any status updates.
